### ISSUE TITLE
[AdaptiveCards- Action.Submit]: Ensures 'More Option' list items has an accessible name

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8664]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.Submit.html
OS Version: 23H2(OS Build 25915.1000)
Browser Version: 117.0.2045.1 Edge Dev
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on Edge Dev browser.
2. Press tab key and navigate to the 'Action.Submit' option and press enter key to select it.
3. Press tab key and navigate to the 'More Option' button under 'Please confirm your data'.
4. Press enter key and activate it.
5. Run the tool and observe the issue.

**Actual Result:**
Options under 'More Item' list item does not have an accessible name.

**Expected Result:**
Options under 'More Option' list items should have an accessible name.

**Element path:**
.ac-popup

**Snippet:**
<div class="ac-ctrl ac-popup" role="listbox"><span role="menuitem" aria-current="false" class="ac-ctrl ac-ctrl-dropdown-item" tabindex="0">Edit data</span><span role="menuitem" aria-current="false" class="ac-ctrl ac-ctrl-dropdown-item" tabindex="0">Delete data</span></div>

**How to fix:**
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

**User Impact:**
It will impact the screen reader user as they will not able to know about the name of component if they don't have an accessible name.

**WCAG Reference:**
**https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html**

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![WCAG4 1 2-Ensures every ARIA input field has an accessible name](https://github.com/microsoft/AdaptiveCards/assets/95913748/710513ec-8dc4-4c55-a133-71c14daf96b1)

![WCAG4 1 2_Ensures every ARIA input field has an accessible name](https://github.com/microsoft/AdaptiveCards/assets/95913748/d7548b30-541b-4b62-8258-30e8ec6a910c)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @jwoo-msft 

As this issue is still repro, and this bug is marked as closed so could you please let us know whether your team will work on this or not also it would be great if please provide us reason for marking this as 'Won't Fix' so that in future we can avoid to logging this type of issue.

**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/ef8a530c-5d4c-4888-ae3f-c89d1ccca86c)

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/user-attachments/assets/c6ed5c97-1716-4014-b063-d3b21e241855)
